### PR TITLE
Add issue templates for this repo

### PR DESCRIPTION
**Pull request by awvwgk**
- bug report template
- feature request template

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: unconfirmed
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
```
Comment (fabothch) [inline .github/ISSUE_TEMPLATE/bug_report.md]: And what behaviour you would actually expect.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+**To Reproduce**
+Steps to reproduce the behaviour:
+1. happens with input (include input files)
+2. start `xtb` with (all the options here)
+3. output showing the error
```
Comment (fabothch) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Please start  `xtb` with (all your options) and additionally "--verbose" and attach this ouput and all other outputs showing the error.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (fabothch) [reply .github/ISSUE_TEMPLATE/bug_report.md]: OK ignore! It follows below

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is.
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
+A clear and concise description of any alternative solutions or features you've considered.
+
```
Comment (fabothch) [inline .github/ISSUE_TEMPLATE/feature_request.md]: ** If possible state how you can assist in providing data or code to to implement the feature**
A  short description of what you can provide to make the feature possible.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (awvwgk) [reply .github/ISSUE_TEMPLATE/bug_report.md]: sounds good.

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 3 above)
Comment (awvwgk) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Sounds like an invitation for a pull request.

**Comment by awvwgk** [comment]
squash & merge next time, this keeps the commit history clean

**Pull request merged**